tck/src/main/java/ee/jakarta/tck/data/framework/read/only/AsciiCharacter_.java

```diff
@@ -15,14 +15,14 @@
  */
 package ee.jakarta.tck.data.framework.read.only;
 
-import jakarta.data.model.Attribute;
-import jakarta.data.model.StaticMetamodel;
+import jakarta.data.metamodel.Attribute;
+import jakarta.data.metamodel.StaticMetamodel;
 
 @StaticMetamodel(AsciiCharacter.class)
 public interface AsciiCharacter_ {
-    public static final Attribute id = Attribute.get();
-    public static final Attribute hexadecimal = Attribute.get();
-    public static final Attribute isControl = Attribute.get();
-    public static final Attribute numericValue = Attribute.get();
-    public static final Attribute thisCharacter = Attribute.get();
+    Attribute id = Attribute.get();
+    Attribute hexadecimal = Attribute.get();
+    Attribute isControl = Attribute.get();
+    Attribute numericValue = Attribute.get();
+    Attribute thisCharacter = Attribute.get();
```

Comment:
This is fine to remove the `public static final` which is implicit for interface fields.  However, with your changes elsewhere in this pull, the `Attribute.get()` part of the code won't compile anymore because you removed that method.  This makes it difficult for me to understand what your intent is with this pull.  If it was just a mistake that `get` and `init` were lost, they will need to be put back and you'll need to figure out how `init` works without the `AttributeInfo` which was removed.  If it was intentional that `get` and `init` are removed, that leaves me wondering what the user actually puts here.  They cannot leave it blank because the field is implicitly final, being on an interface.  If they put `null` there, it will never have a value because the field is final.  Hopefully you do not want the user to directly reference generated code here.